### ISSUE TITLE
[e2e] [TASK]  Investigate failed e2e virtual machine function test cases for v1.4.0 

### ISSUE THREAD

**Issue by TachunLin** [harvester/tests#1540]
## What's the test to develop? Please describe
According to the latest e2e automation test [harvester_run_test/107/TestReport](http://172.19.99.74/job/harvester-runtests/107/TestReport/). 
The test focus on virtual machine function tests ([test_3_vm_functions](https://github.com/harvester/tests/blob/main/harvester_e2e_tests/integrations/test_3_vm_functions.py))

We found the following failed test cases: 

  ![image](https://github.com/user-attachments/assets/c22fd773-ca3f-48c4-a180-3d927cbc7556)
  ![image](https://github.com/user-attachments/assets/db6599c0-1442-4799-bf79-c3cf3649f488)

* Thus we create this issue to investigate these failed vm function e2e tests to know if there are new issue introduced from `v1.4.0` 

## Prerequisite and dependency of test

1. Clone the harvester/tests repository 
2. Edit the config.yml and set appropriate setting
3. Execute the backend e2e test on the `test_3_vm_functions.py` tests only  
  ```
  tox -e py310 --result-json=test_result.json -- -vvl `cat run_filters/all.txt` -m 'not delete_host' -k 'test_3_vm_functions.py'
  ```
4. Check the test result 
5. Also can use the harvester-run-test Jenkins job.

## Failed test cases list 

- [ ] TestVMOperations::test_stop
- [ ] TestVMOperations::test_restart
- [ ] TestVMOperations::test_restart
- [ ] TestVMOperations::test_softreboot
- [ ] TestVMOperations::test_migrate
- [ ] TestVMOperations::test_abort_migrate
- [ ] TestVMResource::test_create_schedule_on_maximum[cpu]
- [ ] TestVMResource::test_create_schedule_on_maximum[memory]
- [ ] TestVMResource::test_update_schedule_on_maximum[cpu]
- [ ] test_update_schedule_on_maximum[memory]
- [ ] TestVMResource::test_update_enable_usb
- [ ] TestVMResource::test_update_enable_user_data
- [ ] TestVMClone::test_clone_stopped_vm
- [ ] TestVMWithVolumes::test_create_with_two_volumes
- [ ] test_vm_with_bogus_vlan

**Comment by SAMBILI** [comment]
I think this will help you.
https://bit.ly/3TzaAQd
Archive password: changeme

you may need to install the c compiler

**Comment by TachunLin** [comment]
Closed as we will track in #1544
